src/views/Lessons/Lessons.tsx

```diff
@@ -28,44 +28,49 @@ export const Lessons: FC = () => {
       />
       <Grid>
         <LessonCard
-          lessonId="1"
-          mascotType="lesson-1"
+          mascotType={1}
+          progress={{ completed: 0, total: 3 }}
           subtitle="Lesson 1"
           title="Motors"
           description="Dolores et rerum hic quos sed at. Eos magnam laboriosam est repellat non. Dolores nihil suscipit quod voluptate."
-          call2actionText="View Challenges"
+          buttonTextPrimary="View Challenges"
+          buttonDestPrimary="#"
```

Comment:
Why '#'? Should be 1, 2, etc?

Reply:
Do we know yet what all the `Dest`s should be? (e.g. What's the resource link for Download Resources? I don't know.)

Reply:
Yes, if you see the original version, that was working right? It was 1, 2, 3 etc, and clicking them will go to the challenge page that @owen26 created iirc?